### PR TITLE
Upgrade Cypress to 6.2.1

### PR DESCRIPTION
**Pull request by ddzz**
## Description
For [this task](https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/13849) we're upgrading to Cypress 6.2.1, which is the latest version. This PR bumps the version and fixes two breaking changes.

## Testing done
- Cypress tests were run locally (in the browser and headless) to verify there are no configuration issues and to catch any breaking changes.
- Tests pass consistently on both Jenkins and CircleCI.

## Acceptance criteria
- [ ] Tests pass on CI
- [ ] Tests can be run locally

### CODE REVIEW

File: src/platform/site-wide/user-nav/tests/e2e/00-required.cypress.spec.js [hunk 1]
```diff
@@ -135,7 +135,7 @@ describe('Site-wide Search functionality with typeahead enabled', () => {
       .children()
       .should('have.length', 5);
     cy.get('[data-e2e-id="sitewide-search-submit-button"]').focus();
-    cy.get('#suggestions-list').should('not.be.visible');
+    cy.get('#suggestions-list').should('not.exist');
```
Comment (pjhill) [inline src/platform/site-wide/user-nav/tests/e2e/00-required.cypress.spec.js:138]: I haven't dug into the commit history, but this must be a new test.  Hooray for swapping out a [deprecated approach for the new one](https://docs.cypress.io/guides/references/migration-guide.html#Non-existent-element-assertions).

**Review by pjhill — APPROVED** [review]

File: src/platform/site-wide/user-nav/tests/e2e/00-required.cypress.spec.js (hunk 1 above)
Comment (holdenhinkle) [reply src/platform/site-wide/user-nav/tests/e2e/00-required.cypress.spec.js:138]: I ran the tests locally twice in headless mode (`yarn cy:run`), and both times the following two tests failed:

- `src/applications/facility-locator/tests/e2e/mobile.cypress.spec.js`
- `src/applications/personalization/view-representative/tests/view-representative.cypress.spec.js`

I'll try running the tests in Cypress ^4.8.0 to compare...

Comment (holdenhinkle) [reply src/platform/site-wide/user-nav/tests/e2e/00-required.cypress.spec.js:138]: Those same two tests failed in Cypress 4.8.0 too so the issue with them doesn't seem to be Cypress 6.2.1.

You stated you ran the tests in Jenkins and CircleCI and they passed so I'm okay to approve.

**Review by holdenhinkle — APPROVED** [review]
LGTM

**Review by theodur — APPROVED** [review]

**Review by rjohnson2011 — APPROVED** [review]
Great work!

**Pull request merged**